docs/js/inject-header.js

```diff
@@ -0,0 +1,135 @@
+// Inject shared XFEL documentation header
+(function() {
+  // For local testing uncomment the line below to use relative paths and
+  // comment out the Github URL.
+  // const baseUrl = '';
+  const baseUrl = 'https://raw.githubusercontent.com/European-XFEL/EXtra/refs/heads/header/docs';
```

Comment:
Needs to be changed to point to `master` before merging:
```suggestion
  const baseUrl = 'https://raw.githubusercontent.com/European-XFEL/EXtra/refs/heads/master/docs';
```